apps/editing-toolkit/editing-toolkit-plugin/event-countdown-block/event-countdown.js

```diff
@@ -13,9 +13,16 @@
 			continue;
 		}
 
-		// parse date into unix time
 		const dtstr = eventDateElem[ 0 ].innerHTML;
-		const eventTime = new Date( dtstr ).getTime();
+
+		let eventTime;
+		if ( ! Number.isNaN( dtstr ) ) {
```

Comment:
Explanation: Basically `! Number.isNaN( dtstr )` means `isUnixTimestamp`.

Reply:
There is an issue with this checking. It breaks existing blocks.
![image](https://user-images.githubusercontent.com/1287077/145981729-7f9824e7-492e-4a9e-8656-bc098ed1a631.png)

Reply:
Perhaps it is okay to use this: `/^[0-9]*$/.test( dtstr );`

![image](https://user-images.githubusercontent.com/1287077/145984216-2a648f9a-474d-4ff9-95c8-00c0e3577e22.png)

Reply:
I can verify that this regex test makes the existing blocks render without breaking. :)

Reply:
Oh man, this is because:

```sh
> Number.isNaN('2022-01-T00:00:00')
false

> isNaN('2022-01-T00:00:00')
true
```

Reply:
One of them JS quirks? Hahaha. :)